portfolio.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles.css" />
+    <link rel="stylesheet" href="./font-awesome-4.7.0/css/font-awesome.min.css">
+    <title>Portfolio</title>
+</head>
+<body>
+  <div id="image-container">
+    <img id="ellipses" src="./images/Top-circles.png" alt="circles">
+  </div>
+  <nav>
+    <p class="placeholder">John Doe</p>
```

Comment:
- [ ] Would you please wrap your logo in an `<a>` tag to be like `<a>John Doe</a>`. This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.